app/views/pages/index.html.erb

```diff
@@ -29,20 +29,24 @@
 
 <% explanation_petitions do |actioned| %>
   <section aria-labelledby="response-threshold-heading" id="petitions-with-response">
-    <div class="threshold-panel response-threshold">
-      <h2 id="response-threshold-heading"><%= Site.formatted_threshold_for_response %></h2>
-      <p>If a petition gets <%= Site.formatted_threshold_for_response %> signatures, the government will&nbsp;respond</p>
-    </div>
+    <%= link_to( petitions_path(state: :with_response) ) do %>
```

Comment:
This needs some more attention. If we are wrapping the threshold panels with a link, then the link should become the threshold panel. This will mean that we apply the hover and focus bg colour changes to the link, not the div